cmd/mount.go

```diff
@@ -111,9 +112,8 @@ be interacting with the file system.`)
 		CacheClock:                 timeutil.RealClock(),
 		BucketManager:              bm,
 		BucketName:                 bucketName,
-		LocalFileCache:             false,
```

Comment:
is this intended?

Reply:
Well, LocalFileCache is always false and setting this to false is a no-op. So, I was trying to remove it but I guess, it could be done as part of a separate PR. So reverted it.